docs/operations/database.md

```diff
@@ -91,7 +91,7 @@ For example:
 ```yaml
 services:
   postgres:
-    image: postgres:16
+    image: postgres:17
```

Comment:
Shouldn't it be 13?

Reply:
In this case, the version is part of a snippet we show in the docs. Here I'd much rather promote users to pick a newer version of Postgres, rather than the minimum support one.